scripts/selector-declaration-builder.js

```diff
@@ -20,11 +29,51 @@ function getQueryableObjectType(services) {
   return dtsObjectTypeAlias(QUERYABLE_TYPE, properties);
 }
 
+function extract$wComments(services) {
+  const $wService = find$wService(services);
+  const $wOperation = $wService.operations.find(
+    operation => operation.name === $W
+  );
+  const [$w$wMembersComment] = $wService.operations
+    .filter(o => o.name === $W)
```

Comment:
use `find`?

Reply:
i am mapping the filter results, so i prefer to use the filter here